fastlane/lib/fastlane/actions/register_devices.rb

```diff
@@ -75,6 +75,7 @@ def self.available_options
                                      env_name: "REGISTER_DEVICES_TEAM_ID",
                                      code_gen_sensitive: true,
                                      default_value: CredentialsManager::AppfileConfig.try_fetch_value(:team_id),
+                                       default_value_dynamic: true,
```

Comment:
wrong indentation level

Reply:
The other lines are indented wrongly, but I didn't want to mix different changes. So I planned to do another PR that fixes them.

Reply:
ah ok